cypcore/Ledger/Graph.cs

```diff
@@ -30,18 +30,17 @@ namespace CYPCore.Ledger
 {
     public interface IGraph
     {
-        Task<VerifyResult> TryAddBlockGraph(BlockGraph blockGraph);
-        Task<VerifyResult> TryAddBlockGraph(byte[] blockGraphModel);
         Task<Transaction> GetTransaction(byte[] txnId);
         Task<IEnumerable<Block>> GetBlocks(int skip, int take);
         Task<IEnumerable<Block>> GetSafeguardBlocks();
         Task<ulong> GetHeight();
         Task<BlockHash> GetHash(ulong height);
+        AsyncAgent<BlockGraph> BlockGraphAgent { get; }
     }
 
     public sealed class Graph : IGraph
     {
-        public const uint BlockmaniaTimeSlot = 0x0000007;
+        public const uint BlockmaniaTimeSlot = 0x000000A;
```

Comment:
Is there a reason to define this in hex?
Maybe including the unit (secs) in the variable name gives some more clarify to what this variable really does.

Reply:
Theres no other reason not to display the numbers in a base 16 numeral system. 
Defining/including the secs in the variable name will definitely change overtime.   As we are still tweaking the sweet spot of the time slot.  So it's better the have it as a constant.